src/Pages/EventManager/EventManager.js

```diff
@@ -57,15 +57,18 @@ export default function EventManager(props) {
   return (
     <div className='event-background'>
       <Header {...headerProps} />
-      <Container>
+      <Container className='event-list'>
         <Button className='create-event' onClick={toggleNewEvent}>
           New Event
         </Button>
         {modal && (
           <EventManagerModal
             modal={modal}
             toggle={toggle}
-            handleDelete={event => deleteEvent(event, props.user.token)}
+            handleDelete={event => {
+              deleteEvent(event, props.user.token);
+              window.location.reload();
```

Comment:
why do we need to reoad the page? its better if we just remove the event from the array that is rendered in `deleteEvent`

Reply:
If we only delete the rendered event from the array, the administrator loses the ability to scroll up and down the page and would have to refresh the page in order to scroll again.